AdminUi/apps/admin_ui/lib/home/identity_details/identity_details.dart

```diff
@@ -0,0 +1,256 @@
+import 'dart:io';
+
+import 'package:admin_api_sdk/admin_api_sdk.dart';
+import 'package:admin_api_types/admin_api_types.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:intl/intl.dart';
+
+class IdentityDetails extends StatefulWidget {
+  final String address;
+
+  const IdentityDetails({required this.address, super.key});
+
+  @override
+  State<IdentityDetails> createState() => _IdentityDetailsState();
+}
+
+class _IdentityDetailsState extends State<IdentityDetails> {
+  Identity? _identityDetails;
+  late List<TierOverview>? _tiers;
```

Comment:
late + nullable is an indicator for wrong usage of nullable or wrong usage of late